packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -420,6 +427,25 @@ contract SystemConfig is OwnableUpgradeable, ISemver, IGasToken {
         emit ConfigUpdate(VERSION, UpdateType.GAS_LIMIT, data);
     }
 
+    /// @notice Updates the EIP-1559 parameters of the chain. Can only be called by the owner.
```

Comment:
This looks good to me! One additional consideration is if we want to do something in `initialize` like:

```solidity
if (eip1559Denominator == 0 && eip1559Elasticity == 0) {
  _setEIP1559Params(canyonDefaults);
}
```

This would sync the L1 state of the system config with the L2 values that are being used

Reply:
No strong feelings either way, but slight preference to leaving it 0, mostly because less code?